include/xrpl/protocol/Feature.h

```diff
@@ -55,6 +55,10 @@
  *    `VoteBehavior::DefaultYes`. The communication process is beyond
  *    the scope of these instructions.
  *
+ * 5) A feature marked as Obsolete can mean either:
+ *    1) It is in the ledger (marked as Supported::yes) and it is Retired now
```

Comment:
they are not "Retired" yet, they are about to become retired. The difference is visible if you scroll to the bottom of `Feature.cpp` file; there are not `feature*` or `fix*` identifiers for a retired feature and there is instead a `[[deprecated]]` `retired*` name, so they cannot be referred to in the code.

Reply:
Makes sense, thanks, Bronek!